### ISSUE TITLE
[BUG] Selection in HoldBrowser

### ISSUE THREAD

**Issue by ThiNei2l** [MoAlyousef/fltk-rs#401]
Hello,
i have an issue with selections in a `HoldBrowser`.
It seems like the last line cannot be selected.
Consider the following example:

```rust
use fltk::{app::*, window::*, browser::*};

fn main() {
    let app = App::default();
    let mut wind = Window::new(100, 100, 400, 300, "Will panic!");

    let mut brw = HoldBrowser::default()
        .with_pos(50, 50)
        .with_size(300, 100);
    brw.add("One");
    brw.add("Two");
    brw.add("Three");

    brw.select(0);
    assert!(!brw.selected(0));
    brw.select(1);
    assert!(brw.selected(1));
    brw.select(2);
    assert!(brw.selected(2));
    brw.select(3);
    assert!(brw.selected(3)); // This will panic

    wind.end();
    wind.show();

    app.run().unwrap();
}
```
The first three assertions behave like expected:
i can select lines 1 and 2, but not 0 since lines start at 1.

But line 3 cannot be selected it seems.

I varied this a bit and the problem is always the last line.

Maybe there is an index shift missing somewhere?

**Comment by MoAlyousef** [comment]
Hello
Yes it was a logic bug in the BrowserExt::select() method. Thanks for pointing it out. I'll publish a fix in a few minutes

**Comment by MoAlyousef** [comment]
I have published the fix in version 0.12.2

**Comment by ThiNei2l** [comment]
Works perfectly.
Thank you for the fast response!
